fossunited/volunteering/doctype/event_volunteer/event_volunteer.json

```diff
@@ -58,18 +61,56 @@
    "fieldname": "city",
    "fieldtype": "Link",
    "label": "City",
-   "options": "City",
-   "reqd": 1
+   "options": "City"
   },
   {
    "fieldname": "identification",
    "fieldtype": "Data",
    "label": "I am"
+  },
+  {
+   "fieldname": "status_info_section",
+   "fieldtype": "Section Break",
+   "label": "Status Info"
+  },
+  {
+   "fieldname": "status",
+   "fieldtype": "Select",
+   "label": "Status",
+   "options": "\nSelected\nWaitlisted\nRejected"
+  },
+  {
+   "fieldname": "volunteer_details_section",
+   "fieldtype": "Section Break",
+   "label": "Volunteer Details"
+  },
+  {
+   "default": "0",
+   "fieldname": "affirmation_for_commitment",
+   "fieldtype": "Check",
+   "label": "Affirmation for commitment"
```

Comment:
? What's this

Reply:
A commitment checkpoint - where they affirm they are ready to give the proposed time commitment

Reply:
This is sort of like an agreement that they've read through and agree to our given guidelines of time commitment and work-group expectations - just like how we gave for CFPs - about reading through talk guidelines